flashlight/fl/dataset/datasets.h

```diff
@@ -17,6 +17,7 @@
 #include "flashlight/fl/dataset/MergeDataset.h"
 #include "flashlight/fl/dataset/PrefetchDataset.h"
 #include "flashlight/fl/dataset/ResampleDataset.h"
+#include "flashlight/fl/dataset/SpanDataset.h"
```

Comment:
clang format to alphabetize?

Reply:
huh, I ran clang-format. is that captured in the repo's .clang-format file?

Reply:
should be, but if you can it, dw about it